pgt/src/PGT/Output/Test/Comments.hs

```diff
@@ -74,9 +87,12 @@ parseComment = do
 
       char <- Text.peekChar'
 
-      unless (Char.isDigit char || char == 'E')
+      unless (Char.isDigit char || Foldable.any (char ==) acceptedChar)
         $ (Err.error . convert . ("expected row count or error text such as (1 row) or (ERROR) but found: (" <>))
         =<< parseLineChars
+      where
+        acceptedChar :: [Char.Char]
```

Comment:
__suggestion: (nitpick)__ Would this be slighly more precise if it returned `Set Char.Char`?

Reply:
Yes, I agree. `NSet Char.Char` improves clarity